yoyodyne/datasets.py

```diff
@@ -599,16 +609,11 @@ def decode_features(
         """
         # Masking source vocab.
         indices = torch.where(
-            (indices >= self.features_idx) | (indices < len(self.special_idx)),
+            (indices >= self.features_idx) | (indices == self.pad_idx),
```

Comment:
Technically the second boolean check is unnecessary. It saves a few flops but don't think it's major in the scheme of things.

Reply:
Good call, removed it.